### PR TITLE
Display error message if failed to load data V2

### PR DESCRIPTION
**Pull request by BrainStone**
Improvement on #28 

- Closes #28

### CODE REVIEW

**Comment by lucko** [comment]
```
Luck - Today at 19:06
any reason for re-creating the readPage function
it seems a little unnecessary to me
```

**Comment by BrainStone** [comment]
It's a mini shortcut. I can remove it if you want

**Comment by lucko** [comment]
Yeah, please. :)

**Comment by BrainStone** [comment]
There we go.

File: assets/app.js [hunk 1]
```diff
@@ -877,12 +876,12 @@ function loadFromParams(params) {
...
                 $.getJSON(rawUrl, function(permsData) {
                     loadData(permsData)
-                })
+                }).fail(showLoadingError)
             } else {
                 var permsData = JSON.parse(fileObject.content)
                 loadData(permsData);
             }
-        })
+        }, showLoadingError)
```
Comment (lucko) [inline assets/app.js]: This one doesn't use `.fail(...)` ??

**Comment by BrainStone** [comment]
My bad! That is in fact wrong. Will fix it

**Comment by lucko** [comment]
🎉

**Pull request merged**